packages/integration-tests/src/tests/console/sso-connectors/sso-connectors-test-cases.ts

```diff
@@ -0,0 +1,38 @@
+// Will extend this type definition later since we are going to configure the SSO connectors with specific values.
+export type SsoConnectorTestCase = {
+  connectorName: string;
+  connectorFactoryName: string;
+};
+
+const microsoftEntraID: SsoConnectorTestCase = {
+  connectorName: 'Microsoft Entra ID',
+  connectorFactoryName: 'Microsoft Entra ID',
+};
+
+const googleWorkspace: SsoConnectorTestCase = {
+  connectorName: 'Google Workspace',
+  connectorFactoryName: 'Google Workspace',
+};
+
+const okta: SsoConnectorTestCase = {
+  connectorName: 'Okta',
+  connectorFactoryName: 'Okta',
+};
+
+const oidc: SsoConnectorTestCase = {
+  connectorName: 'OIDC',
+  connectorFactoryName: 'OIDC',
+};
+
+const saml: SsoConnectorTestCase = {
+  connectorName: 'SAML',
+  connectorFactoryName: 'SAML',
+};
```

Comment:
Since they are all the same, could merge into one single property

Reply:
Updated.